### ISSUE TITLE
Accessibility issues on CSV uploader screens

### ISSUE THREAD

**Issue by zdeveloper** [CDCgov/prime-simplereport#4386]
## Area or Component

- Upload spreadsheet screen
- View upload history screen
- CSV template guide screen
- Submission details screen

## Accessibility Violation

- Pages must have at least h1
- Invalid `aria-describe` on the file selector in the upload screen
